content/software-security/selecting-a-base-image.md

```diff
@@ -0,0 +1,83 @@
+
+## Choose an image based on a software distribution that prioritizes package update frequency and security hygiene
+
+The packages inside a container are often sourced from a Linux distribution such as Debian, Ubuntu, or [Wolfi](https://edu.chainguard.dev/open-source/wolfi/overview/). The update frequency and hence the security of an image therefore also depends on the software development practices associated with that distribution. Software teams should therefore assess the distribution underlying the packages in their images. All things equal, software teams should prefer a distribution that prioritizes frequent updates so that vulnerabilities are quickly fixed.
+
+## Choose an image with maintainers that actively support it
+
+Not all container images are actively maintained. Like open source software packages in general, container images require one or more persons to ensure timely and frequent rebuilds, to manage broader security concerns, and to respond to bugs and issues. Unmaintained or lightly maintained images can pose a problem to software teams that want to prioritize development velocity and security. Without the ability to rapidly work with upstream image maintainers, software teams must either accept the risk of using old versions of the image that potentially haveunpatched bugs, or consider creating and maintaining a fork with the overhead associated with that course of action.
+
+## Choose an image that is smaller
+
+Smaller images (fewer MB) means less storage costs and less transmission costs. When an image is pulled thousands of times a day, even a small difference in size can quickly add to the cost.
+
+## Choose an image that is signed
+
+Software signing ensures that attackers that tamper with a software artifact, such as a container image, can be detected. Software teams should choose to use container images that have been signed, with tools such as [Cosign](https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-sign-a-container-with-cosign/), and verify the signatures to ensure that the images on which they depend are tamper-free.
+
+## Choose an image with an SBOM
+
+A [software bill of materials](https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) (SBOM), a list of ingredients for a piece of software, provides visibility into the components of a piece of software, enabling transparency and informed decision-making. Choosing base images with SBOMs generated by the party responsible for building the artifact helps the image consumer better make decisions related to the security and health of the code.
+
+## Self-assessment
+
+[True or False] The frequency at which a container image is rebuilt does not affect the number of known software vulnerabilities associated with the container image.
```

Comment:
Love that you added assessment here - thanks for trying it out! 

I think the code below should work (at least it did locally). Do you want to try adding it to both of your questions and seeing if it renders correctly in the preview? Otherwise, you might want to think of another way of providing the answer . . . or we can leave assessment out until we have the formatting worked out. 

<details>
  <summary>
   [True or False] The frequency at which a container image is rebuilt does not affect the number of known software vulnerabilities associated with the container image.
  </summary>
John Speed's answer and quick explanation. 
</details>

Reply:
@sheesh, thanks for figuring out the proper syntax. I tried in the latest commit. Let me check it out in the preview :)

Reply:
I need help getting this page to show up.

When I look in the preview, I can't even see the page. Sad. I think it's a small TOML change. I just tried something. Any help is welcome :)

Reply:
@ltagliaferri or @sheesh , any thoughts?

Reply:
nvm, it worked! Just making one last tweak.